contribute/developer-guide.md

```diff
@@ -76,7 +76,10 @@ When you log in for the first time, Grafana asks you to change your password.
 
 #### Building on Windows
 
-The Grafana backend includes Sqlite3 which requires GCC to compile. So in order to compile Grafana on Windows you need to install GCC. We recommend [TDM-GCC](http://tdm-gcc.tdragon.net/download).
+The Grafana backend includes SQLite which requires GCC to compile. So in order to compile Grafana on Windows you need to install GCC. We recommend [TDM-GCC](http://tdm-gcc.tdragon.net/download). Eventually, if you use [Scoop](https://scoop.sh), you can install GCC through that.
```

Comment:
Change to: The Grafana backend includes SQLite which requires GCC to compile. As a result, to compile Grafana on Windows, you must intsall GCC. We recommend using [TDM-GCC](http://tdm-gcc.tdragon.net/download). If you use [Scoop](https://scoop.sh), you can install GCC through that.

Reply:
Why is "We recommend using TDM-GCC" better than "We recommend TDM-GCC"? I don't think it reads any better.